UnityProject/Assets/Scripts/Doors/FireLock.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+
+public class FireLock : InteractableDoor, IServerLifecycle
+{
+	private MetaDataNode metaNode;
+
+	public override void TryClose()
+	{
+	}
+
+	public override void TryOpen(GameObject performer)
+	{
+
+	}
+
+	public void TickUpdate()
+	{
+		if (!Controller.IsClosed)
+		{
+			if (metaNode.GasMix.Pressure < AtmosConstants.MINIMUM_OXYGEN_PRESSURE)
```

Comment:
The doors need to be more sensitive to pressure changes.

![Uj52N3CunD](https://user-images.githubusercontent.com/4493127/80377455-39d43500-8893-11ea-9c3e-acbc9965bef4.gif)


Try using this constant instead:
```suggestion
			if (metaNode.GasMix.Pressure < AtmosConstants.WARNING_LOW_PRESSURE)
```

Reply:
I think it would be good if they also checked if it's greater than WARNING_HIGH_PRESSURE/HAZARD_HIGH_PRESSURE too.